### PR TITLE
Fix additional cases in labels handling

### PR DESCRIPTION
**Pull request by tiraboschi**
**What this PR does / why we need it**:
Fix additional cases in labels handling on operands, add more unit tests.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
https://issues.redhat.com/browse/CNV-38579
```

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: controllers/operands/dashboard_test.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ import (
 	"strings"
 	"time"
 
+	hcoutil "github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
```
Comment (nunnatsa) [inline controllers/operands/dashboard_test.go:11]: please re-group imports

**Review by nunnatsa — APPROVED** [review]
/approve

File: controllers/operands/dashboard_test.go (hunk 1 above)
Comment (tiraboschi) [reply controllers/operands/dashboard_test.go]: done

**Pull request merged**